### PR TITLE
Change page list labels (private, etc) to be gray

### PR DESCRIPTION
**Pull request by nheagy**
Refs #11863

> Yellow "private" text should be gray, preceded by a •

Before | After
----- | -----
| image | image |

To test:
- Open the pages list, find or create a private page
- Ensure that the new appearance matches the description

Update release notes:

- [ ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Pages/PageListTableViewCell.m [hunk 1]
```diff
@@ -145,27 +144,22 @@ - (void)configureBadges
...
-            badgesString = [badgesString stringByAppendingString:@" · "];
-        }
-        badgesString = [badgesString stringByAppendingString:NSLocalizedString(@"Local changes", @"Title of the Local Changes Badge")];
+        [badges addObject:NSLocalizedString(@"Local changes", @"Title of the Local Changes Badge")];
     }
     
-    self.badgesLabel.text = badgesString;
-}
-
-- (void)configureTimeStamp
```
Comment (danielebogo) [inline WordPress/Classes/ViewRelated/Pages/PageListTableViewCell.m]: This needs to be removed also in `- (void)setPost:(AbstractPost *)post` where `[self configureTimeStamp]`

**Comment by nheagy** [comment]
@danielebogo thanks! Fixed that up 👍

**Comment by mattmiklic** [comment]
Looking good!

**Comment by mattmiklic** [comment]
Actually one small thing; instead of `•` could you use the `·` character? That's Unicode U+00B7. That would be the more typographically correct symbol. :)

**Comment by nheagy** [comment]
Thanks @danielebogo @mattmiklic!

**Pull request merged**